tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -1713,7 +1713,6 @@ qos/test_qos_sai.py:
     conditions:
       - "asic_type in ['barefoot'] and topo_name in ['t1']"
       - "topo_type in ['m0', 'mx']"
-      - "asic_type in ['vs']"
```

Comment:
Did any of the tests pass?
If not, you can set vs to xfail and observe for a while before deciding whether to completely remove the skip condition.